### DISCUSSION TITLE
What is the 'iss' value in request parameters?

### DISCUSSION THREAD

**Discussion by VitSens** [keycloak/keycloak#25551] [Q&A] [answered]
I am using version 23.0.1 of Keycloak and have observed the parameter iss={redirect_url}. What does it mean, and can I disable it?

Example:
![image](https://github.com/keycloak/keycloak/assets/55030208/c37cf093-8510-4dea-aa25-2000cb6f11f1)

**Comment by jonkoops** [comment, marked as answer]
Looks related to #22486. If you have reproducible example, please log this as a bug. Also like mentioned in https://github.com/keycloak/keycloak/issues/22486#issuecomment-1713521165 you might have to use a more recent version of Keycloak JS.
